app/Http/Controllers/MemberInductionController.php

```diff
@@ -34,73 +35,53 @@ function __construct(PolicyRepository $policyRepository, UserRepository $userRep
     }
 
     /**
-     * Display a listing of the resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function index()
-    {
-        $users = $this->userRepository->getPendingInductionConfirmation();
-        return \View::make('account.induction.index')->withUsers($users);
-    }
-
-    /**
-     * Action the admin approve requests
-     *
-     * @param $id
-     *
-     * @return mixed
-     * @throws \BB\Exceptions\AuthenticationException
-     */
-    public function approve($id)
-    {
-        $user = User::findWithPermission($id, 'comms');
-
-        if (\Input::has('inducted_by')) {
-            $user->inducted_by = \Auth::id();
-
-            $user->save();
-
-            \Notification::success('Updated');
-        }
-
-        return \Redirect::route('account.induction.index');
-    }
-
-    /**
-     * Display the specified resource.
+     * Show the peer induction page
      *
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function show($id)
-    {
-        $user = User::findWithPermission($id);
-
-        $document = $this->policyRepository->getByName('member-agreement');
-
-        $htmlDocument = Markdown::defaultTransform($document);
+    public function show(Request $request)
+    { 
+        $user = false;
+        if(!\Auth::guest()){
+            $user = \Auth::user();
+        }
+        $induction_code = Settings::get("general_induction_code");
+        $prefill_code = $request->has('code') ? $request->input('code') : '';
 
-        return view('account.induction.show')->with('user', $user)->with('document', $htmlDocument);
+        return view('general-induction.show')
+            ->with('user', $user)
+            ->with('general_induction_code', $induction_code)
+            ->with('prefill_induction_code', $prefill_code);
     }
 
     /**
-     * Update the specified resource in storage.
+     * Set a peer induction.
      *
      * @param  \Illuminate\Http\Request  $request
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, $id)
+    public function update(Request $request)
     {
-        $user = User::findWithPermission($id);
+        $input = $request->only('rules_agreed', 'inductee_email', 'induction_code');
+        
+        $this->inductionValidator->validate($input);
 
-        $input = $request->only('rules_agreed', 'induction_completed');
+        $induction_code = Settings::get("general_induction_code");
 
-        $this->inductionValidator->validate($input);
+        if(strtolower($input['induction_code']) != strtolower($induction_code)){
```

Comment:
Could be worth a trim to guard against folks accidentally including spaces (phones love to add a extra whitespace at the end sometimes)

```suggestion
        if(trim(strtolower($input['induction_code'])) != strtolower($induction_code)){
```